config/db

```diff
@@ -50,36 +49,36 @@ rbx_1.2.3_patch_level=20110315
 rbx_1.2.4_patch_level=20110705
 rbx_url=http://asset.rubini.us
 rbx_2.0.0_url=http://releases.rubini.us
-rubinius_repo_url=git://github.com/rubinius/rubinius.git
+rubinius_repo_url=https://github.com/rubinius/rubinius.git
 ree_version=1.8.7
 ree_configure_flags=--dont-install-useful-gems
 ree_1.8.6_patch_level=20090610
 ree_1.8.6_url=http://rubyforge.org/frs/download.php/58677
```

Comment:
This doesn't exist anymore. The only working place I could find is http://mirror.as24220.net/pub/rubygems/files/emm-ruby/ (sadly, https:// not supported).